src/proxy/src/main/java/ee/ria/xroad/proxy/protocol/ProxyMessage.java

```diff
@@ -111,8 +112,9 @@ public SoapFault getFault() {
      */
     public String getSoapContentType() {
         return isMimeEncodedSoap() || hasAttachments()
-                ? (originalContentType != null ? originalContentType : encoder.getContentType())
-                : MimeUtils.TEXT_XML_UTF8;
+                ? originalContentType != null
+                    ? originalContentType : encoder.getContentType()
+                : MimeTypes.TEXT_XML_UTF8;
     }
```

Comment:
Was this broken? Is the new version easier to read? This seems like it should have at least one less ternary operator.